.dockerignore

```diff
@@ -20,7 +20,6 @@ vendor/bundle
 /tmp
 .DS_Store
 .swp
-.approvals
```

Comment:
should we keep it? This one once this is merged people won't have an unknown `.approvals` file they need to manually delete and might accidentally re-commit or re-submit into a PR

Reply:
That's fair! I re-added this back into `.dockerignore` in the last commit :)